new-backoffice/extending/development-flow/vite-package-setup.md

```diff
@@ -62,7 +64,9 @@ export default defineConfig({
 });
 ```
 
-This alters the Vite default output into a **library mode**, where the output is a JavaScript file with the same name as the `name` attribute in `package.json`. The name is `my-extension` if you followed this tutorial with no changes.
+This alters the Vite default output into a **library mode**, where the output is a JavaScript file with the same name as the `name` attribute in `package.json`. The name is `my-dashboard` if you followed this tutorial with no changes.
+
+The source code that is compiled lives in the `src` folder of your package folder. Here you can see a `my-element.js` file. You can confirm that this file is the one specified as our entry on the Vite config file that we just created.
```

Comment:
Part of our styleguide is [to not use "just" ](https://docs.umbraco.com/welcome/contribute/style-guide#editorializing) word, therefore this would be my suggestion:
```suggestion
The source code that is compiled lives in the `src` folder of your package folder and that is where you can see a `my-element.js` file. You can confirm that this file is the one specified as our entry on the Vite config file that we recently created.
```